[ADD] email marketing: essentials, mailing lists, unsubscriptions (#4121)
odoo/documentation: content/applications/marketing/email_marketing.rst

> :nosearch:

Reviewer: When a page has children pages, it isn't visible by default. If you add the "essentials" in this page, you'd need these options:
```suggestion
:show-content:
:show-toc:
```

.. image:: email_marketing/graph-view.png
   :align: center
   :alt: How the Graph view appears in the Odoo Email Marketing application.

In the upper-left corner, there is a :guilabel:`Measures` drop-down, providing even more filter
options to further customize the graph views.

Those :guilabel:`Measures` options are: :guilabel:`A/B Testing percentage`, :guilabel:`Color
Index`, and :guilabel:`Count` (which is selected, by default).

Filters, Group By, and Favorites search options
-----------------------------------------------

Regardless of the view chosen for the :guilabel:`Email Marketing` dashboard, the
:guilabel:`Filters`, :guilabel:`Group by`, and :guilabel:`Favorites` options are always available.

These options provide various ways to specify and organize the information seen on the
:guilabel:`Email Marketing` dashboard.

.. tabs::

   .. tab:: Filters

      This drop-down provides different ways to filter email information on the dashboard. The
      options are: :guilabel:`My Mailings`, :guilabel:`Sent Date`, :guilabel:`Archived`, and
      :guilabel:`Add Custom Filter`. If :guilabel:`Add Custom Filter` is selected, Odoo reveals an

>       additional drop-down, with three customizable fields to fill in, and Odoo reveals the
>       appropriate results that fit that specific criteria.

Reviewer 2: ```suggestion
      additional drop-down menu, with three customizable fields to fill in, in order to retrieve results
      that fit more specific criteria.
```
